src/pages/development/backward-incompatible-changes/highlights.md

```diff
@@ -11,14 +11,22 @@ This page highlights backward-incompatible changes between Adobe Commerce and Ma
 
 The following major backward-incompatible changes were introduced in the 2.4.6 Adobe Commerce and Magento Open Source releases:
 
+*  New default value for automatic redirects
 *  New system configuration for customer segments
 *  New system configuration for limiting products in grid
 *  New system configuration for OpenSearch module
 *  Symfony dependencies upgraded to latest LTS version
 *  Zend_Filter replaced with laminas-filter
 *  Zend_HTTP replaced with laminas-http
+*  Zend_Json replaced with laminas-json
 *  Zend_Validate replaced with laminas-validator
 
+### New default value for automatic redirects
+
+To improve performance, the default value for `generate_category_product_rewrites` in the [`app/code/Magento/CatalogURLRewrite/etc/config.xml`](https://github.com/magento/magento2/blob/2.4-develop/app/code/Magento/CatalogUrlRewrite/etc/config.xml#L12) file was changed from `1` to `0`. This change disables [automatic redirects](https://experienceleague.adobe.com/docs/commerce-admin/marketing/seo/url-rewrites/url-redirect-product-automatic.html) by default.
+
+The new default does not change existing records in the `catalog_url_rewrite_product_category` and `url_rewrite` database tables when upgrading to 2.4.6, but no new rewrites are added. However, using the `bin/magento config:set catalog/seo/generate_category_product_rewrites` command to change this configuration setting after upgrading to 2.4.6 permanently deletes all rewrites with no ability to restore them. This may cause unresolved "category/product" type URL conflicts that you must resolve by manually updating URL keys.
+
```

Comment:
We should add suggestion that if the generate_category_product_rewrites functionality is wanted for them,  they should manually set this value, and it will work with same functionality as before.

Here's example of what I think we should add.

> If `Generate "category/product" URL Rewrites` functionality is still wanted, it is recommended to manual set this configuration option to enabled, such as `bin/magento config:set catalog/seo/generate_category_product_rewrites 1`, then behaviour will stay the same after upgrade.